### PR TITLE
Change response from plain-text to JSON

### PR DESCRIPTION
**Pull request by riseremi**

### CODE REVIEW

File: server/src/com/rovioli/storascore/Main.kt [hunk 1]
```diff
@@ -25,6 +26,11 @@ fun Application.main() {
     // For each GET header, adds an automatic HEAD handler (checks the headers of the requests
     // without actually getting the payload to be more efficient about resources)
     install(AutoHeadResponse)
+    install(ContentNegotiation) {
+        gson {
+            setPrettyPrinting()
```
Comment (dmitvitalii) [inline server/src/com/rovioli/storascore/Main.kt]: Cool!

Comment (riseremi) [reply server/src/com/rovioli/storascore/Main.kt]: Thank you.

File: server/src/com/rovioli/storascore/response/ScoreResponse.kt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package com.rovioli.storascore.response
+
+/**
+ * by riseremi on 08/11/2018
+ * riseremi@icloud.com
+ */
+// TODO: add score output
+data class ScoreResponse(val score: Int)
```
[the remark this answers is not in the document]
Comment (riseremi) [reply server/src/com/rovioli/storascore/response/ScoreResponse.kt]: Yeah, I just don't know what needs to be in the score response. Can you propose a change?

[the remark this answers is not in the document]
Comment (dmitvitalii) [reply server/src/com/rovioli/storascore/response/ScoreResponse.kt]: You can have a look at the `ScoreDao` and see what will be returned there.
For now, supposedly `/highscores` returns an array of scores:
```
[
  {
    "name": "riseremi",
    "score": 20001
  },
  {
    "name": "dmitvitalii",
    "score": 10001
  },
  {
    "name": "Barak",
    "score": 2016
  }
]
```
_(it's a bug if not)_

**Comment by riseremi** [comment]
There's something new! Please, check it out.

**Pull request merged**